src/components/CodeSnippet/CodeSnippet.astro

```diff
@@ -262,6 +262,40 @@ function parseMarkingDefinition(serializedArr: string, parts: RegExp, parseError
 					}
 				}
 			}
+
+			.copy-button {
+				position: absolute;
+				top: 0.5rem;
+				right: 0.5rem;
+				cursor: pointer;
+				border-radius: 0.4rem;
+				opacity: 0.2;
+				-webkit-transition: visibility 0s, opacity 0.3s linear;
+				-moz-transition: visibility 0s, opacity 0.3s linear;
+				-o-transition: visibility 0s, opacity 0.3s linear;
+				transition: visibility 0s, opacity 0.3s linear;
```

Comment:
We don’t need prefixes for these properties and I think we don’t need the `visibility` property here?

```suggestion
				transition: opacity 0.3s linear;
```